sklego/mixture/bayesian_gmm_detector.py

```diff
@@ -162,11 +165,11 @@ def score_samples(self, X):
         if len(X.shape) == 1:
             X = np.expand_dims(X, 1)
 
-        return self.gmm_.score_samples(X) * -1
+        return self.gmm_.score_samples(X)
 
     def decision_function(self, X):
         # We subtract self.offset_ to make 0 be the threshold value for being an outlier:
-        return self.score_samples(X) + self.likelihood_threshold_
+        return self.score_samples(X) - self.offset_
```

Comment:
Both this detector and gmm one required some (breaking changes) to comply to the tests and how scikit expects `score_samples`, `decision_function` and `predict` to interoperate